UseCase.md

```diff
@@ -1,64 +1,83 @@
-# BTCPay Server Use Case
+# BTCPay Server Use Cases
+  
+Build with the community in mind, BTCPay server is a feature-rich software with plenty of use-cases that can solve problems for different types of users. 
 
-While primarily build for merchants, BTCPay server is feature-rich software which has plenty of use-cases and solves problems for different users. In this guide, we will show you some of the use-cases of BTCPay. The usage of the software is not limited to these groups.
+In this guide, we will show you some of the use-cases of BTCPay, but it is by no means limited to the groups of users we mentioned.
+
+So let's have a look at what you can do with BTCPay and what are the benefits of using it.
 
 ## Merchants
 
-Benefits of using BTCPay as a merchant:
+By choosing BTCPay to process payments, you are:
 
 * Saving money (no fees, no subscriptions)
-* Payments go directly to your wallet (No third-party)
-* Enhanced privacy for you and your customers
-* Easy integration and installation
-* Self-sovereignty 
+* Cutting off the middle-man (Payments go directly to your wallet)
+* Enhancing privacy for you and your customers (no address re-use, no IP leaks to third parties)
+* Saving time (easy integration and installation)
+* Protecting yourself from interference in your business (self-sovereignty) 
+
+To enjoy most of these benefits, you don't even need to run a BTCPay server yourself, you could just create an account on someone else's server. It will be even easier to set up and entirely free, at the cost of relying on the server admin to keep it functional and up-to-date.  
```

Comment:
> It will be even easier to set **up and entirely free**, at the cost of relying on the server admin to keep it functional and up-to-date.

1. Not all hosts are free.
2. I wouldn't encourage the third-party hosting like this as people will get a false sense of security with server admin, but he can always close the server and leave them in the dark.

Suggestion remove this part 

> It will be even easier to set up and entirely free, at the cost of relying on the server admin to keep it functional and up-to-date.